planet_scope/ndvi_difference/README.md

```diff
@@ -0,0 +1,39 @@
+---
+title: NDVI difference between two dates
+parent: Planetscope
+grand_parent: Planet
+layout: script
+permalink: /planet_scope/ndvi_difference/
+nav_exclude: true
+scripts:
```

Comment:
Same as the max ndvi readme. It would be nice to have the `eob.js` and the `raw.js`. You can find the script I created for Sentinel-2 ndvi difference ([eob.js](https://github.com/sentinel-hub/custom-scripts/blob/main/sentinel-2/ndvi_difference/eob.js) and [raw.js](https://github.com/sentinel-hub/custom-scripts/blob/main/sentinel-2/ndvi_difference/raw.js)).

Reply:
I think this is too complicated as we are using multiple acquisitions in this instance. In the difference script there is a before and after. For precedent the Sentinel-2 version of this script does not have an eob version.